src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -93,6 +93,14 @@ $replicasetdetails-warning: orange;
   font-size: 0.85em;
 }
 
+.md-button.kd-replicasetdetail-sidebar-editpods-button {
```

Comment:
Do not prefix with .md-button. Just use class that you've defined. This is a best practice to always just use classes with names that describe object's function. With that you're not coupled with HTML structure, nor with the implementation (md-button here).